Flank/src/main/java/com/walmart/otto/filter/TestFilters.java

```diff
@@ -45,7 +45,9 @@ private static TestFilter parseSingleFilter(String argument) {
     boolean isNegate = matcher.group(1) != null;
 
     Collection<String> args =
-        Arrays.stream(matcher.group(3).split(",")).collect(Collectors.toSet());
+        Arrays.stream(matcher.group(3).split(","))
+            .map(String::trim) //splitting by comma will leave leading or trailing spaces
```

Comment:
again, I prefer consider this a parsing error so we support less cases, but I guess it's up to you.